bookwyrm/templates/ostatus/error.html

```diff
@@ -0,0 +1,76 @@
+{% load i18n %}
+
+{% block content %}
+<div class="block">
+    {% if error == 'invalid_username' %}
+    <div class="notification is-warning has-text-centered" role="status">
+        {% blocktrans %}
```

Comment:
The whitespace on these trans blocks is going to cause formatting issues in the locale file. I think it makes more sense to translate each sentence separately, with as little html in each as possible

Reply:
I've pulled them into separate strings but we need to keep _some_ html inside blocktrans otherwise it will be impossible to translate properly into languages that don't share the same grammar and word order as English.

Reply:
yeah for sure - your change looks perfect, thank you